src/kernel/arch/x86/pic.zig

```diff
@@ -220,14 +220,16 @@ pub fn spuriousIrq(irq_num: u8) bool {
 }
 
 pub fn setMask(irq_num: u16) void {
```

Comment:
I probably should have mentioned this in the PIC PR, but could `clearMask` and `setMask` be merged since they are very similar?

Reply:
I have seen this as one function, but like to separate for a clear distinction instead of having another parameter for switching between the two. What do u think

Reply:
Yeah on second thoughts having two is fine. An alternative would be to make them wrappers around a function that does take the extra parameter, but the compiler will probably extract the common code from them anyway.